src/js/data/rolesData.json

```
@@ -10,6 +10,12 @@
         "product": "Labs",
         "displayText": "Labs"
     },
+    "labs-label":{
+        "description": "Neo4j Labs project label",
+        "labelCategory": "product",
```

```suggestion
        "labelCategory": "content",
```

I don't think this makes any difference to the handling or output at the moment, but I think this should match the existing `labs` label. (We are using labels to provide some semantic information to content, not for this type of label yet, but we might do in future).

---

ok, I was just wondering if it had any impact. I can revert the change no problem